compiler/src/iree/compiler/Codegen/Dialect/LoweringConfig.cpp

```diff
@@ -278,10 +278,24 @@ void setTranslationInfo(IREE::HAL::ExecutableExportOp exportOp,
 // operations.
 // ===----------------------------------------------------------------------===//
 
+Operation *getLoweringConfigCarryingOp(ArrayRef<Operation *> computeOps) {
+  for (Operation *op : computeOps) {
+    if (getLoweringConfig(op)) return op;
+  }
+  return nullptr;
```

Comment:
Echo of comment from below. Maybe be better to avoid passing `nullptr`s around

Reply:
Done.